test/ex_machina_test.exs

```diff
@@ -171,14 +131,15 @@ defmodule ExMachinaTest do
     user = MyApp.ExMachina.create(:user, admin: true)
 
     assert user == %{id: 3, name: "John Doe", admin: true}
-    assert_received {:created, %{name: "John Doe", admin: true}}
+    assert_received {:custom_save, %{name: "John Doe", admin: true}}
   end
 
   test "create/2 builds factory and performs custom save if repo is not set" do
-    record = MyApp.NonEctoFactories.create(:foo)
+    record = MyApp.ExMachina.create(:user)
 
-    assert record == %{foo: :bar}
-    assert_received {:custom_save, %{foo: :bar}}
+    created_record = %{admin: false, id: 3, name: "John Doe"}
+    assert record == created_record
+    assert_received {:custom_save, ^created_record}
   end
 
   test "create/2 raises a helpful error if no repo and no custom function" do
```

Comment:
replace "if no repo and no custom function" with "if no custom save_record/1 function is defined" or something like that?